app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -859,9 +859,10 @@ class Settings(private val appContext: Context) : PreferencesHolder {
     /**
      * Indicates if the jump back in CRF should be shown.
      */
-    var shouldShowJumpBackInCFR by booleanPreference(
+    var shouldShowJumpBackInCFR by lazyFeatureFlagPreference(
         appContext.getPreferenceKey(R.string.pref_key_should_show_jump_back_in_tabs_popup),
-        default = true
+        featureFlag = FeatureFlags.showJumpBackInCFR,
+        default = { onboardScreenSection[OnboardingSection.JUMP_BACK_IN_CFR] == true },
```

Comment:
is `onboardScreenSection[OnboardingSection.JUMP_BACK_IN_CFR]` returning a boolean? if so should this just be 
```suggestion
        default = onboardScreenSection[OnboardingSection.JUMP_BACK_IN_CFR],
```

Reply:
I think we ought to keep the current default function, as the signature for `lazyFeatureFlagPreference` requires `() -> Boolean` and `onboardScreenSection[OnboardingSection.JUMP_BACK_IN_CFR]` returns a `Boolean?`.

This same approach can be seen multiple times in `Settings.kt`.

Reply:
Gotcha gotcha. Yeah, if it's a nullable boolean, I agree with your approach, since the `== true` will safeguard us in that situation. Otherwise, you'd have to do something like:

`onboardScreenSection[OnboardingSection.JUMP_BACK_IN_CFR] ?: false` 

which could be arguably be just as ambiguous.